src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -84,2270 +84,9 @@ use Google\Tagger\V1\AddTagRequest;
 use Google\Tagger\V1\LabelerGrpcClient;
 
 /**
- * Service Description: This API represents a simple digital library.  It lets you manage Shelf
- * resources and Book resources in the library. It defines the following
- * resource model:
- *
- * - The API has a collection of [Shelf][google.example.library.v1.Shelf]
- *   resources, named ``bookShelves/*``
- *
- * - Each Shelf has a collection of [Book][google.example.library.v1.Book]
- *   resources, named `bookShelves/&#42;/books/*`
- *
- * Check out [cloud docs!](https://cloud.google.com/library/example/link).
- * This is [not a cloud link](http://www.google.com).
- *
- * Service comment may include special characters: <>&"`'&#64;.
- *
- * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
- * more frequently than those which have been declared beta or 1.0, including changes which break
- * backwards compatibility.
- *
- * This class provides the ability to make remote calls to the backing service through method
- * calls that map to API methods. Sample code to get started:
- *
- * ```
- * try {
- *     $libraryServiceClient = new LibraryServiceClient();
- *     $shelf = new Shelf();
- *     $response = $libraryServiceClient->createShelf($shelf);
- * } finally {
- *     $libraryServiceClient->close();
- * }
- * ```
- *
- * Many parameters require resource names to be formatted in a particular way. To assist
- * with these names, this class includes a format method for each type of name, and additionally
- * a parse method to extract the individual identifiers contained within names that are
- * returned.
- * @experimental
+ * {@inheritdoc}
```

Comment:
This is a phpdoc tag that says to use the documentation from the parent class, in this case the client implementation.